core/listings.go

```diff
@@ -43,6 +43,8 @@ const (
 	MaxCountryCodes          = 255
 	EscrowTimeout            = 1080
 	SlugBuffer               = 5
+
+	DefaultCoinDivisibility uint32 = 10e7
```

Comment:
Suppose to be 100_000_000 instead of 10_000_000

Reply:
It is 100,000,00 but I can change it to 1e8 instead of 10e7 if that's more readable.